src/components/dev-dot-content/index.tsx

```diff
@@ -88,6 +88,7 @@ const DevDotContent = ({
 		content = (
 			<MDXRemote
 				{...mdxRemoteProps}
+				// @ts-expect-error -- The next-mdx-remote types are wrong here, we don't want a record of ReactNode
```

Comment:
We're on an old version of `next-mdx-remote` with incorrect types, so where I couldn't make updates in this repo I added this comment.